documentation/src/main/asciidoc/user_guide/cross_site_replication.adoc

```diff
@@ -201,6 +201,30 @@ Both the local and the backup call(if the "backupFailurePolicy" is set to "FAIL"
 ===== Asynchronous local cluster
 In the case of asynchronous local clusters, the backup data is sent during the commit phase. If the backup call fails and the "backupFailurePolicy" is set to "FAIL" then the user is notified through an exception.
 
+==== Expiring cache entries across sites
+You can configure {brandname} to expire entries, which controls the amount of
+time entries remain in the cache.
+
+* `lifespan` expiration is suitable for cross-site replication because entries
+can expire on either site without the need to synchronize system time between
+sites.
+
+* `max-idle` expiration is not suitable for cross-site replication because
+{brandname} needs to send requests across sites to determine if cache entries
+have reached the idle timeout. This also requires sites to have synchronized
+system times regardless of time zone.
```

Comment:
I am not sure if we need to clarify the time zone bit here.

But to be honest we may want to be more clear and say that max-idle just doesn't work when used in cross site. For example if an entry expires via max idle in one site, it doesn't ask the other site if it has read that variable recently, it just expires it. Thus it can remove an entry that was recently accessed.